tests/test_util.py

```
@@ -887,6 +887,30 @@ def test_complex_input(self):
+    def test_superscripts_and_vulgar_fractions_do_not_crash(self):
+        from ovos_number_parser import extract_number
+        for lang in ("pt", "es", "ca", "it", "gl", "an", "ast", "mwl", "oc", "ro"):
+            for text in ("¹", "²", "³", "½"):
+                with self.subTest(lang=lang, text=text):
+                    self.assertFalse(extract_number(text, lang=lang))
+
+    def test_arabic_indic_digits_still_work(self):
+        from ovos_number_parser import extract_number
+        for lang in ("pt", "es", "ca", "it", "gl", "an", "ast", "mwl", "oc", "ro"):
+            with self.subTest(lang=lang):
+                self.assertEqual(extract_number('١', lang=lang), 1)
```

Ruff (0.15.21)

[warning] String contains ambiguous `١` (ARABIC-INDIC DIGIT ONE). Did you mean `l` (LATIN SMALL LETTER L)?

(RUF001)